.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx elm-format src --yes
```

Comment:
Should this run with npm so we keep version within project? Or does that not work in this context?

Reply:
hmmmm, well I thought `npx` used the locally available option first.

https://docs.npmjs.com/cli/v7/commands/npx

I ran `npx elm-format --help` and discovered version `0.8.5` was being used. But when I ran the same command via the package.json scripts I get the same result.  So now I'm not sure if our npm set up is reliable either :0